test/libsolidity/semanticTests/various/code_length_contract_member.sol

```diff
@@ -0,0 +1,17 @@
+// Test to see if type.code.length does extcodesize(type) only when type is an address.
+struct S {
+    bytes32 code;
+    bytes32 another;
+}
+
+contract C {
+    S s;
+
+    function f() public returns (uint, uint, bool) {
+        return (s.code.length, s.another.length, address(this).code.length > 150);
```

Comment:
Huh, didn't even knew FixedBytes has `.length`. Do we have tests for that? :)

Reply:
Yes. `test/libsolidity/semanticTests/array/fixed_bytes_length_access.sol`